### PR TITLE
newsboat: update to 2.21.

### PR DESCRIPTION
**Pull request by daniel-eys**
Also removed some lines in `post_install()` which are already handled by `make install`.

### CODE REVIEW

**Comment by Piraty** [comment]
do you care that some test fail?
```
test cases:   495 |   492 passed | 3 failed
assertions: 22478 | 22472 passed | 6 failed

=> ERROR: newsboat-2.21_1: do_check: 'TERM=$TERM TMPDIR=/dev/shm ./test' exited with 6
=> ERROR:   in do_check() at srcpkgs/newsboat/template:23
=> ERROR: newsboat-2.21_1: do_check: '( cd test && TERM=$TERM TMPDIR=/dev/shm ./test )' exited with 1
=> ERROR:   in do_check() at srcpkgs/newsboat/template:23
```

File: srcpkgs/newsboat/template [hunk 1]
```diff
@@ -25,8 +25,4 @@ do_check() {
 
 post_install() {
 	vlicense LICENSE
-	vsconf doc/example-bookmark-plugin.sh bookmark-plugin.sh
```
Comment (Piraty) [inline srcpkgs/newsboat/template:28]: please double-check the lines you removed from `post_install`,
for example: `diff <(xbps-query -Rf newsboat) <(fd -t f . usr | sort | sed 's/^/\//')`

**Review by Piraty — CHANGES_REQUESTED** [review]
.

File: srcpkgs/newsboat/template (hunk 1 above)
Comment (daniel-eys) [reply srcpkgs/newsboat/template:28]: This file is already at `/usr/share/doc/newsboat/examples/example-bookmark-plugin.sh`.
I made sure that no actual files were lost.

**Comment by daniel-eys** [comment]
> do you care that some test fail?
> 
> ```
> test cases:   495 |   492 passed | 3 failed
> assertions: 22478 | 22472 passed | 6 failed
> 
> => ERROR: newsboat-2.21_1: do_check: 'TERM=$TERM TMPDIR=/dev/shm ./test' exited with 6
> => ERROR:   in do_check() at srcpkgs/newsboat/template:23
> => ERROR: newsboat-2.21_1: do_check: '( cd test && TERM=$TERM TMPDIR=/dev/shm ./test )' exited with 1
> => ERROR:   in do_check() at srcpkgs/newsboat/template:23
> ```

Strange, all tests actually passed on my machine.
```
All tests passed (22485 assertions in 495 test cases)
```
2.21 is running without any issues so far, so I guess this should be fine.

File: srcpkgs/newsboat/template (hunk 1 above)
Comment (Piraty) [reply srcpkgs/newsboat/template:28]: i dind't mean that file in specific but the others. my diff has about 20 changed lines. please run my snippet (in `destdir/newsboat-...`)

**Comment by Piraty** [comment]
> Strange, all tests actually passed on my machine.
> 
> ```
> All tests passed (22485 assertions in 495 test cases)
> ```
> 
> 2.21 is running without any issues so far, so I guess this should be fine.

did you build with `XBPS_CHECK_PKGS=yes` in etc/conf ?

File: srcpkgs/newsboat/template (hunk 1 above)
Comment (daniel-eys) [reply srcpkgs/newsboat/template:28]: I diffed the package contents. The `contrib` folder is currently present twice in the package, once installed by `make install` to `/usr/share/doc/newsboat/contrib` and then again by the `vcopy` to `/usr/share/examples/newsboat/contrib`. That's I was trying to fix.

**Comment by daniel-eys** [comment]
> > Strange, all tests actually passed on my machine.
> > ```
> > All tests passed (22485 assertions in 495 test cases)
> > ```
> > 
> > 
> > 2.21 is running without any issues so far, so I guess this should be fine.
> 
> did you build with `XBPS_CHECK_PKGS=yes` in etc/conf ?

Yes

**Comment by Minoru** [comment]
> do you care that some test fail?

I am Newsboat's maintainer, so I do :) Can I view the whole output somewhere? I don't see tests being run in the Travis logs.

**Comment by Piraty** [comment]
Tests fail when building in an x86_64-musl environment. Running tests is neither part of travis nor buildbot.

Reproduce:
```
./xbps-src zap && ./xbps-src binary-bootstrap x86_64-musl
./xbps-src -j 1 -Q pkg newsboat
```
Full log: https://termbin.com/yuj5 (might not be available after a while)

**Comment by Minoru** [comment]
Thanks! We've had a few reports about problems with musl builds, but unfortunately never gotten to the bottom of the matter. I copied your output to newsboat/newsboat#1216, maybe someone will look at it after all ._.

**Pull request closed without merging**